### DISCUSSION TITLE
Add window offset

### DISCUSSION THREAD

**Discussion by akwebb1** [Ulauncher/Ulauncher#907] [Ideas]
It would be nice to have an X axis offset option for the positioning of the search window when it pops up.  I use ulauncher under Windows WSL2 with a hotkey to launch it.  Since I have 2 monitors the search box shows up directly in the center, half on one monitor and half on the other since my WSL2 instance with the X server sees my dual monitors as a large single monitor.  If I could offset the launch position right or left by half the virtual monitor size ulauncher would be a perfect companion for WSL2.

**Comment by akwebb1** [comment]
This discussion can be closed.  This was an old request and I moved on to another launcher since it was unusable in my scenario.

**Reply by friday** [reply]
Just unclick the arrow up symbol below the post and unsubscribe from notification to show you no longer are interested in this @akwebb1. There's no way to close these.

**Reply by sarim** [reply]
> This discussion can be closed. This was an old request and I moved on to another launcher since it was unusable in my scenario.

@akwebb1 Can you share which launcher you moved on to? I'm also looking for a launcher to use in WSL2

**Reply by Wielding** [reply]
I am no longer a Windows user.
